### ISSUE TITLE
Error trying to get login/register page

### ISSUE THREAD

**Issue by devhubpl** [LaravelDaily/Larastarters#92]
I am getting an error trying to go on register on login pages:
`"Unable to locate a class or view for component [auth-validation-errors]."`
All steps I've done: install fresh laravel 9, install larastarter, run install script (breeze with windmill) and run npm
```
composer create-project laravel/laravel larastarter
composer require laraveldaily/larastarters --dev
php artisan larastarters:install
npm install && npm run dev

```
My env: PHP 8.1.2 and Laravel 9.35.1
What I do wrong?

**Comment by PovilasKorop** [comment]
@devhubpl recently Laravel Breeze changed their component names, and made some structural changes - https://github.com/laravel/breeze/commit/dc3564d69d103ca8de29754022c0b74fa17602fc - so we need to fix it on our end now, will do it soon.

**Comment by PovilasKorop** [comment]
@devhubpl  fixed with the new release of `v2.1.0`, please update
